CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+7.0.0 / 2020-01-22
+===================
+- Added logic for the is_element_visible function which reduces implicit wait when no wait parameter is passed to the
+function
```

Comment:
Please keep each bullet point on a single line - you can ignore Pycharm's line length suggestion for documentation files.